numpy/core/src/private/ufunc_override.c

```diff
@@ -106,7 +105,10 @@ PyUFunc_WithOverride(PyObject *args, PyObject *kwds,
     }
 
     for (i = 0; i < nargs + nout_kwd; ++i) {
-        PyObject *method;
+        PyObject *obj;
+        int j;
```

Comment:
This can move into the `if (with_override ...)` block - no need to declare it here